README.md

```diff
@@ -61,14 +61,15 @@ import {
   hotkeyKeyUX,
   jumpKeyUX,
   focusGroupKeyUX,
+  overrides,
```

Comment:
Let’s add second word to the name to avoid possible conflict with user code.

Like `overrides` → `hotkeyOverrides`